schema.sql

```diff
@@ -7,6 +7,27 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL
+    species_id INT,
+    owners_id INT,
+    PRIMARY KEY (id),
+    CONSTRAINT fk_species
+    FOREIGN KEY(species_id) 
+	REFERENCES species(id),
+    CONSTRAINT fk_owners
+    FOREIGN KEY(owners_id) 
+	REFERENCES owners(id)
```

Comment:
When Making changes to a table, you should use the `ALTER TABLE` command. This way you wouldn't have to delete the table and re-create again. You did write altering commands in the `data.sql` file, so I suggest you revert this to how it was before.